lib/oneview-sdk/resource/api800/c7000/resource.rb

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright 2018 Hewlett Packard Enterprise Development LP
```

Comment:
What's the use of this file ? 
Inheriting from base resource file and there is no methods in it.